src/main/java/com/google/alpollo/AuthenticationServlet.java

```diff
@@ -1,16 +1,37 @@
 package com.google.alpollo;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
-import com.google.appengine.api.users.UserService;
+import com.google.gson.Gson;
 
+/** Servlet that returns an auth info that includes status whether the user logged in and link to auth. */
 @WebServlet("/auth")
 public class AuthenticationServlet extends HttpServlet {
+    private final Gson gson = new Gson();
+
+    /** Class provides info about logged in status of the user and link to the authentication. */
+    private class LoggedInInfo {
+        private final boolean loggedIn;
```

Comment:
nit: it's a good practice to prefix boolean parameters with `has` or `is` whenever it makes sense. I think `isLoggedIn` makes sense here.